src/Reflection/BetterReflection/BetterReflectionProvider.php

```diff
@@ -373,9 +373,17 @@ public function getConstant(Node\Name $nameNode, ?NamespaceAnswerer $namespaceAn
 			if ($resolvedPhpDoc->isDeprecated() && $resolvedPhpDoc->getDeprecatedTag() !== null) {
 				$deprecatedMessage = $resolvedPhpDoc->getDeprecatedTag()->getMessage();
 
-				// filter raw version number messages like in
-				// https://github.com/JetBrains/phpstorm-stubs/blob/9608c953230b08f07b703ecfe459cc58d5421437/filter/filter.php#L478
-				if (Strings::match($deprecatedMessage ?? '', '#^\d+\.\d+(\.\d+)?$#') === null) {
+				$matches = Strings::match($deprecatedMessage ?? '', '#^(\d+)\.(\d+)(?:\.(\d+))?$#');
```

Comment:
this duplicates the logic which should already have happened in BetterReflection..  I think it is wrong